RELEASE-NOTES.md

```diff
@@ -1,5 +1,91 @@
 # Prefect Release Notes
 
+## Release 2.12.0
+
+### Introducing `Flow.serve()`
+
+We're excited to introduce a radically simple way to deploy flows. 
+
+The new `.serve()` method available on every flow allows you to take your existing flows and schedule or trigger runs via the Prefect UI and CLI. 
```

Comment:
Should we actually detail what it means to "serve" your flows?

Reply:
I tried to detail that with

> allows you to take your existing flows and schedule or trigger runs via the Prefect UI and CLI.

Let me know if I can make that clearer.

Reply:
I think it could be but I'll defer to @billpalombi

Reply:
Yeah I think thats great